Assets/MRTK/SDK/Experimental/PulseShader/Scripts/SurfacePulseOnSelect.cs

```diff
@@ -0,0 +1,290 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.SurfacePulse
+{
+    /// <summary>
+    /// Script for generating pulse shader effect on the surface with air-tap gesture event.
+    /// </summary>
+    [AddComponentMenu("Scripts/MRTK/SDK/SurfacePulse")]
+    public class SurfacePulseOnSelect : MonoBehaviour, IMixedRealityPointerHandler
```

Comment:
A diff shows almost entirely shared code between this and `SurfacePulse`. Could this inherit from that class and override where needed?

Reply:
Merged the feature into the original SurfacePulse.cs and deleted SurfacePulseOnSelect.cs